src/shared/cloudformation/activation.ts

```diff
@@ -48,15 +46,106 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
                 getIdeProperties().codelenses
             )
         )
-        getLogger().error('Failed to activate template registry', e)
+        getLogger().error('Failed to activate template registry: %s', e)
```

Comment:
for a long time, error() had the (dubious) feature of printing an `Error` parameter in addition to the message param. I guess that got lost at some point and now these `e` args aren't printed in many places.

We should probably just update all the error() call-sites instead of restoring that "feature".